### ISSUE TITLE
Add configuration option for units which would show up in balloon

### ISSUE THREAD

**Issue by marks** [timwis/vizwit#154]
For example, show `Total: 100 crimes` instead of `Total: 100`

**Comment by timwis** [comment]
Good idea! We could also probably move the balloon contents to template files to more easily make changes like this. (not to scope creep, just made me think of that)
